<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <widget class="QTabWidget" name="MainTabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>750</width>
      <height>525</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="VideoPlayerTab">
     <attribute name="title">
      <string>Demo</string>
     </attribute>
     <widget class="QPushButton" name="PlayVideoButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>10</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
     <widget class="QPushButton" name="FocusInButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>70</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Focus In</string>
      </property>
     </widget>
     <widget class="QFrame" name="VideoFrame">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>645</width>
        <height>485</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QProgressBar" name="VideoProgress">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>2</x>
         <y>480</y>
         <width>640</width>
         <height>4</height>
        </rect>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLCDNumber" name="FPSCounter">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>5</y>
         <width>50</width>
         <height>25</height>
        </rect>
       </property>
       <property name="digitCount">
        <number>3</number>
       </property>
       <property name="value" stdset="0">
        <double>999.000000000000000</double>
       </property>
      </widget>
      <widget class="QLCDNumber" name="PFPSCounter">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>30</y>
         <width>50</width>
         <height>25</height>
        </rect>
       </property>
       <property name="digitCount">
        <number>3</number>
       </property>
       <property name="value" stdset="0">
        <double>999.000000000000000</double>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="PauseVideoButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>40</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="SettingsTab">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QGroupBox" name="VideoStreamSrcGp">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>100</y>
        <width>500</width>
        <height>90</height>
       </rect>
      </property>
      <property name="title">
       <string>Video Stream Source</string>
      </property>
      <widget class="QPushButton" name="VideoFileLoadButton">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>29</y>
         <width>75</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Load Video</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="WebcamChB">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Use Webcam Output</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="AppStyleGroup">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>10</y>
        <width>500</width>
        <height>70</height>
       </rect>
      </property>
      <property name="title">
       <string>Application Style</string>
      </property>
      <widget class="QComboBox" name="AppThemeCoB">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="ThemeL">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>47</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Theme:</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="VideoStreamG">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>210</y>
        <width>500</width>
        <height>70</height>
       </rect>
      </property>
      <property name="title">
       <string>Video Stream</string>
      </property>
      <widget class="QComboBox" name="ResolutionCoB">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>240p 4:3 (320×240)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>360p 4:3 (480×360)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>480p 4:3 (640×480)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>720p HD 16:9 (1280×720)</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="ResolutionL">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Resolution:</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="MiscellaneousGp">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>300</y>
        <width>500</width>
        <height>90</height>
       </rect>
      </property>
      <property name="title">
       <string>Miscellaneous</string>
      </property>
      <widget class="QCheckBox" name="FpsChB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Show FPS\PFPS</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="ProgressBarChb">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Show Video Progress Bar</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
